src/_P036_FrameOLED.ino

```diff
@@ -346,20 +395,25 @@ boolean Plugin_036(byte function, struct EventStruct *event, String& string)
         if (isFormItemChecked(F("p036_ScrollLines")))
           Settings.TaskDevicePluginConfigLong[event->TaskIndex][0] += 0x20000;          // Bit 17
 
+        Settings.TaskDevicePluginConfigLong[event->TaskIndex][0] += 0x100000;            // Bit 23-20 = 1 -> version V1
```

Comment:
Please use the logical OR function to set a bit. Addition only works when the bit you want to set is a 0.

Reply:
Done!